### ISSUE TITLE
Add the remaining property into async timeout

### ISSUE THREAD

**Issue by Kentzo** [aio-libs/async-timeout#20]
It's useful to know how much time is remaining when scheduling tasks that either do not support asyncio's mechanics or provide an independent timeout.
One such example would be running a subprocess with a program that can accept a timeout:

```python
async with timeout(1234) as t:
    p = await asyncio.create_subprocess_exec('iptables', '-w', int(t.remaining), ...)
    await p.wait()
```

**Comment by asvetlov** [comment]
Please make a Pull Request.

**Comment by asvetlov** [comment]
Fixed by #21
